### DISCUSSION TITLE
Why Phonon dispersion curve is not smooth???

### DISCUSSION THREAD

**Discussion by vishchandan007** [ttadano/alamode#159] [General]
I am trying to calculate the Phonon dispersion curve for NaBaSb. Here, I am attaching the calculated plot up to the Harmonic level, i.e., NORDER = 1. I am not able to rectify why the plot is not smooth. Please let me know if anyone can help me out.

Regards 
Chandan
![nbs222_HARM](https://github.com/ttadano/alamode/assets/134786103/5bacce4f-7f17-4998-a2d7-4f63ae142827)

**Comment by ttadano** [comment]
Please check if you are correctly using a primitive lattice vectors for the `&cell` in ANPHON.

**Comment by vishchandan007** [comment]
Dear Tadano,

Thank you for your reply. I checked my input files, and based on my knowledge, I think the input files are correct. 
However, I am attaching the input files for NaBaSb, including the POSCAR, input of anphon, and output for your reference. Please let me know if I am making any silly mistakes in the calculation.

Thanks and regards, 
Chandan
IIT Delhi, India

[NBT_input.zip](https://github.com/ttadano/alamode/files/13692601/NBT_input.zip)

**Reply by ttadano** [reply]
Hi,

I've checked your input and found you used the supercell lattice parameters for the `&cell` field in nbs_phband.in. Please use the primitive cell lattice parameters instead.

**Reply by vishchandan007** [reply]
Dear Tadano,

Thank you for your reply. After incorporating your suggestion, I am getting a perfect phonon dispersion curve.

---
Chandan
